WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -3,6 +3,67 @@ local L = WeakAuras.L;
 
 WeakAuras.regionPrototype = {};
 
+-- Adjusted Duration
+
+function WeakAuras.regionPrototype.AddAdjustedDurationToDefault(default)
+  default.useAdjustededMax = false;
+  default.useAdjustededMin = false;
+end
+
+function WeakAuras.regionPrototype.AddAdjustedDurationOptions(options, data, order)
+  options.useAdjustededMin = {
+    type = "toggle",
+    name = L["Set Minimum Value/Duration"],
```

Comment:
Hmm, I think it needs to have a reference to progress in some way. As it's otherwise not clear for what you are setting a minimum. I've changed it ot Set Minimum Progress.